security/pkg/pki/ra/k8s_ra.go

```diff
@@ -73,6 +73,12 @@ func (r *KubernetesRA) Sign(csrPEM []byte, certOpts ca.CertOpts) ([]byte, error)
 		return nil, err
 	}
 	csrName := chiron.GenCsrName()
+	signerFromRA := r.raOpts.CaSigner
+	certSigner := certOpts.CertSigner
```

Comment:
Don't we need to check it is within CertSignerDomain?

Reply:
Added

Reply:
fixed, Thanks!